lib/libzfs/libzfs_sendrecv.c

```diff
@@ -1225,7 +1225,7 @@ send_progress_thread(void *arg)
 	(void) strlcpy(zc.zc_name, zhp->zfs_name, sizeof (zc.zc_name));
 
 	if (!pa->pa_parsable)
-		(void) fprintf(stderr, "TIME        SENT   SNAPSHOT\n");
+		(void) fprintf(stderr, "TIME        SENT   SNAPSHOT [%s]\n", zhp->zfs_name);
```

Comment:
Since there is already a space for delimiting, what benefit does adding '[]' provide?  Or, I think we can do without the '[]'

Reply:
I'm not beholden to brackets encompassing the snapshot name.  I will drop them and force push a new commit.